cookbook/core/extend_flyte/sql.py

```diff
@@ -0,0 +1,79 @@
+"""
+.. _extend_sql_sqlite3:
+
+SQL Query Tasks - example SQLite3
+===================================
+
+Flyte tasks are not restricted to pure containers. This is one of the most important design decisions in Flyte.
```

Comment:
I feel like this is slightly misleading... this is true, but if they dig in a bit more (or if the user looks at running pods on the cluster) they'll soon see that this task does indeed run a container.  I feel it's not great to imply that this task doesn't run a container... and then spin one up anyways, and a container that they didn't even build (that could potentially have malicious code).  could we rephrase to something like
```suggestion
Flyte tasks are not always restricted to running user-supplied containers, nor even containers at all. Indeed, this is one of the most important design decisions in Flyte.
```